README.md

```diff
@@ -10,7 +10,20 @@ as well as a comprehensive set of primitives for building custom caching solutio
 
 ## Getting Started
 
-### TinyLFU Cache
+### Documentation Highlights
+
+#### Custom Keys
+
+Cachemere supports using arbitrary types as keys. The page
+on [Using Custom Keys](https://coveooss.github.io/cachemere/usingCustomKeys.html) covers this in greater detail.
+
+#### Heterogeneous Lookup
+
+Like `std::map`, `cachemere::Cache` supports heterogeneous lookup. Take a look
+at [the documentation on heterogeneous lookup](https://coveooss.github.io/cachemere/heterogeneousLookup.html) for more
```

Comment:
The link `https://coveooss.github.io/cachemere/heterogeneousLookup.html`  doesn't work on my machine: I'm not sure if it's expected or not.